### PR TITLE
fix loging on stagings

### PR DESCRIPTION
**Pull request by mmelko**
add enable-test-support scenario for running on stagings

//test: @stage-smoke

### CODE REVIEW

File: ui-tests/src/test/resources/features/other/enable-test-support.feature [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@disabled
+@enable-test-support
+Feature: Enable test support post syndesis install
+
+  Scenario: enable test support and wait for restart
+    When enable test-support on server pod
+    And sleep for jenkins delay or "5" seconds
+    Then wait for Syndesis to become ready
```
Comment (avano) [inline ui-tests/src/test/resources/features/other/enable-test-support.feature]: add a new line here

File: utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java [hunk 2]
```diff
@@ -162,4 +164,11 @@ public void waitForTodo() {
             fail("Waiting for Todo app was interrupted with exception: " + e.getMessage());
         }
     }
+
+    @When("^enable test-support on server pod$")
+    public void enableTestSupportOnServerPod() throws Throwable {
+        HashMap<String,String> envVars = new HashMap<>();
+        envVars.put("ENDPOINTS_TEST_SUPPORT_ENABLED","true");
+        OpenShiftUtils.xtf().updateDeploymentConfigEnvVars("syndesis-server",envVars);
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java]: add a space before ","

Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java]: both lines

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/other/enable-test-support.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (avano) [inline ui-tests/src/test/resources/features/other/enable-test-support.feature]: newline

File: utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java [hunk 4]
```diff
@@ -162,4 +166,9 @@ public void waitForTodo() {
             fail("Waiting for Todo app was interrupted with exception: " + e.getMessage());
         }
     }
+
+    @When("^enable test-support on server pod$")
+    public void enableTestSupportOnServerPod() {
+        OpenShiftUtils.updateEnvVarInDeploymentConfig("syndesis-server","ENDPOINTS_TEST_SUPPORT_ENABLED", "true");
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java]: space before comma

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**